docs/Install/Running_Swarm_Learning.md

```diff
@@ -78,7 +79,10 @@ NOTE: These options do not apply to the `swarm-learning/bin/stop-swarm` script.
 |`--host-ip <IP address or DNS name>`\(Mandatory parameter\)<br>|The IP address or DNS name of the host system on which this Swarm Learning node is created.| |
 |`--sn-ip <IP address or DNS name>`|The IP address or DNS name of the host system on which the Swarm Network \(SN\) node with which this Swarm Learning node must associate, is running.| |
 |`--sn-api-port <port number>`|Host port for the API Server of the associated Swarm Network node|30304|
+|`--sn-api-service <fqdn>:<port number>`|Fully Qualified Domain Name for the SN API Service of associated SN node. Here, Port number is optional.| |
 |`--sl-fs-port <port number>`|Host port for this Swarm Learning node's File Server.|30305|
+|`--sl-fs-service <fqdn>:<port number>`|Fully Qualified Domain Name for the SL FS Service of associated SL node. Here, Port number is optional.| |
```

Comment:
-> Fully Qualified Domain Name and optional port for this Swarm Learning node's file service. Default: None